war/src/main/js/keyboard-shortcuts.js

```diff
@@ -0,0 +1,25 @@
+import hotkeys from "hotkeys-js"
+
+window.addEventListener("load", () => {
+  const searchBar = document.querySelector("#search-box")
+  searchBar.placeholder = searchBar.placeholder + ` (${translateModifierKeysForUsersPlatform("CMD+K")
+    .replace("CMD", "⌘")})`
+
+  hotkeys(translateModifierKeysForUsersPlatform("CMD+K"), () => {
+    searchBar.focus()
+
+    // Returning false stops the event and prevents default browser events
+    return false
+  })
+})
+
+/**
+ * Given a keyboard shortcut, e.g. CMD+K, replace any included modifier keys for the user's
+ * platform e.g. output will be CMD+K for macOS/iOS, CTRL+K for Windows/Linux
+ * @param {string} keyboardShortcut The shortcut to translate
+ */
+function translateModifierKeysForUsersPlatform(keyboardShortcut) {
+  const useCmdKey = navigator.platform.toUpperCase().indexOf("MAC") >= 0 ||
+    navigator.platform.toUpperCase() === "IPHONE"
```

Comment:
Are iPads iPhones?

Reply:
Just tested this on my devices (running Monterey and iOS/iPadOS 16) -

Mac: `MacIntel`
iPhone: `iPhone` (`MacIntel` when in desktop mode)
iPad: `MacIntel` (`iPad` when in mobile mode)

So not confusing at all 😭😂  

I've added an additional condition just in case a user is on mobile mode on their iPad.

Reply:
> So not confusing at all



<details>
<summary>Wanna guess what's my `navigator.platform` on M1? </summary>

`MacIntel` lol

</details>